From 0d9fb4b04f678add24f81e15602b2d57569730f2 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Sun, 8 Sep 2019 16:01:39 +0300
Subject: [PATCH] board: clearfog: enable both DDR clocks

This is needed to support the Clearfog GTR memory configuration.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 board/solidrun/clearfog/clearfog.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/solidrun/clearfog/clearfog.c b/board/solidrun/clearfog/clearfog.c
index 1742aa8921c9..3fb991f627a3 100644
--- a/board/solidrun/clearfog/clearfog.c
+++ b/board/solidrun/clearfog/clearfog.c
@@ -77,6 +77,7 @@ static struct mv_ddr_topology_map board_topology_map = {
 
 struct mv_ddr_topology_map *mv_ddr_topology_map_get(void)
 {
+	board_topology_map.clk_enable = 0x3;
 	/* Return the board topology as defined in the board code */
 	return &board_topology_map;
 }
-- 
2.23.0

